src/App.js

```diff
@@ -1,28 +1,36 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
+
+import React from 'react';
+import {Route} from 'react-router-dom';
+import CountryDetail from './components/CountryDetail';
+import List from './components/List';
+import 'bootstrap/dist/css/bootstrap.css';
+
+
 import './App.css';
 
-class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
-  }
+
+
+function App() {
+
+  const navStyle = {
+    color: "white",
+    backgroundColor: "#007BFF",
+    fontWeight: "bold",
+  };
+
+  return (
+    
+     <div class="container" >
+     <nav style={navStyle} className="navbar navbar-expand-lg">
+     <a style={navStyle} class="navbar-brand" href="#">Wiki Countries</a>
+     </nav>
+        <div class="row" >
+          <List /> 
+          <Route path='/details/:cc3a' component={CountryDetail}/>
+      </div>      
+    </div>
+  );
 }
 
-export default App;
+//Switch??
```

Comment:
Switch is basically the react-router equivalent to a JavaScript switch statement.

Within the Switch component, Route and Redirect components are nested inside. Starting from the top Route/Redirect component in the Switch, to bottom Route/Redirect, each component is evaluated to be true or false based on whether or not the current URL in the browser matches the path/from prop on the Route/Redirect component.

if you do it without it wont have this true/false ability so you can render more than one route at a time and have to use the exact keyword to get around it :)